{
  "uid" : "ca200d47680dc74e",
  "name" : "TestClass.test_xfail",
  "title" : "Test xfail",
  "time" : {
    "start" : 1460649914844,
    "stop" : 1460649914851,
    "duration" : 7
  },
  "summary" : {
    "attachments" : 0,
    "steps" : 0
  },
  "failure" : {
    "message" : "",
    "stackTrace" : "self = <tests.test_xml.TestClass instance at 0x03400990>\n\n    @pytest.mark.xfail()\n    @allure.story('Story1')\n    def test_xfail(self):\n        'this test is an xfail'\n>       assert False\nE       assert False\n\ntests\\test_xml.py:29: AssertionError"
  },
  "description" : {
    "value" : "\nУ этого модуля, как и у его тестов, кириллические описания, данные в юникоде\n\nCreated on Oct 19, 2013\n\n@author: pupssman\n\r\nthis test is an xfail",
    "type" : "TEXT"
  },
  "severity" : "NORMAL",
  "status" : "PENDING",
  "testId" : null,
  "suite" : {
    "uid" : "983b3ba1607cd63c",
    "name" : "tests.test_nonascii",
    "title" : "Test nonascii"
  },
  "steps" : [ ],
  "attachments" : [ ],
  "issues" : [ ],
  "labels" : [ {
    "name" : "thread",
    "value" : "5548-MainThread"
  }, {
    "name" : "feature",
    "value" : "Feature1"
  }, {
    "name" : "story",
    "value" : "Story1"
  }, {
    "name" : "suite-title",
    "value" : null
  }, {
    "name" : "suite-name",
    "value" : "tests.test_nonascii"
  }, {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "host",
    "value" : "ZinhVcoin-PC"
  } ],
  "parameters" : [ ]
}